packages/amplify-provider-awscloudformation/src/aws-predictions-regions.js

```diff
@@ -124,6 +129,7 @@ const Comprehend = {
   'eu-south-1': 'eu-central-1',
   'ap-northeast-1': latestComprehendRegions >= 1 ? 'ap-northeast-1' : 'us-east-1',
   'ap-northeast-2': latestComprehendRegions >= 1 ? 'ap-northeast-2' : 'us-east-1',
+  'ap-northeast-3': 'ap-northeast-1',
```

Comment:
Comprehend is not available in ap-northeast-3. Falling back to ap-northeast-1.

https://docs.aws.amazon.com/general/latest/gr/comprehend.html